tests/functional/aws-node-sdk/test/bucket/deleteBucketTagging.js

```diff
@@ -0,0 +1,94 @@
+const assert = require('assert');
+const { errors } = require('arsenal');
+const { S3 } = require('aws-sdk');
+const async = require('async');
+
+const getConfig = require('../support/config');
+
+const bucket = 'policyputtaggingtestbucket';
+
+const validTagging = {
+    TagSet: [
+        {
+            Key: 'key1',
+            Value: 'value1',
+        },
+        {
+            Key: 'key2',
+            Value: 'value2',
+        },
+    ],
+};
+
+// Check for the expected error response code and status code.
+function assertError(err, expectedErr, cb) {
+    if (expectedErr === null) {
+        assert.strictEqual(err, null, `expected no error but got '${err}'`);
+    } else {
+        assert.strictEqual(err.code, expectedErr, 'incorrect error response ' +
+            `code: should be '${expectedErr}' but got '${err.code}'`);
+        assert.strictEqual(err.statusCode, errors[expectedErr].code,
+            'incorrect error status code: should be  ' +
+            `${errors[expectedErr].code}, but got '${err.statusCode}'`);
+    }
+    cb();
+}
+
+describe.only('aws-sdk test delete bucket tagging', () => {
+    let s3;
+
+    before(done => {
+        const config = getConfig('default', { signatureVersion: 'v4' });
+        s3 = new S3(config);
+        return done();
+    });
+
+    beforeEach(done => s3.createBucket({ Bucket: bucket }, done));
+
+    afterEach(done => s3.deleteBucket({ Bucket: bucket }, done));
+
+    it('should delete tag', done => {
+        async.series([
+            next => s3.putBucketTagging({
+                AccountId: s3.AccountId,
+                Tagging: validTagging, Bucket: bucket,
+            }, (err, res) => {
+                next(err, res);
+            }),
+            next => s3.getBucketTagging({
+                AccountId: s3.AccountId,
+                Bucket: bucket,
+            }, (err, res) => {
+                next(err, res);
+            }),
+            next => s3.deleteBucketTagging({
+                AccountId: s3.AccountId,
+                Bucket: bucket,
+            }, (err, res) => {
+                next(err, res);
+            }),
+            next => s3.getBucketTagging({
+                AccountId: s3.AccountId,
+                Bucket: bucket,
+            }, (err, res) => {
+                next(err, res);
+            }),
+        ], (err) => {
+            assertError(err, 'NoSuchTagSet', done);
+        });
+    });
+
+    it('should make no change when deleting tags on bucket with no tags', done => {
+        async.series([
+            next => s3.deleteBucketTagging({
+                AccountId: s3.AccountId,
+                Bucket: bucket,
+            }, (err, res) => {
+                next(err, res);
+            }),
+        ], (err) => {
+            assert.ifError(err);
+            done(err);
+        });
+    });
+});
```

Comment:
Same here: "no change" but we are not testing the returned values. Can you do a GET, store the result, then DELETE, and then GET and compare with the first result?

Reply:
You can maybe "shadow" the error by checking that the error is NoSuchtagSet and if so, return a next(null, null), otherwise continue returning next(err, res) to catch a possible error

Reply:
done https://github.com/scality/cloudserver/pull/4481/commits/bc8c6f527ec83cfcc7cb7b2bcf2d83ff9b35609b#diff-76b4a70b2a7470e6cde6671151107006543e8d06514f148d131f7f86ea0007b3R80